docs/guide/accuracy/chain-of-thought.md

```diff
@@ -1,7 +1,4 @@
-<figure markdown>
-  ![Chain-of-Thought prompting](/assets/standardvscotprompt.png){ width="600" }
-  <figcaption>Chain of Thought Prompting (Wei et al)</figcaption>
-</figure>
+![Chain-of-Thought-Prompting](https://raw.githubusercontent.com/JTarakRam/autolabel/main/docs/assets/standardvscotprompt.jpg)
```

Comment:
thanks for suggesting it! I think the chain of thought doc works correctly and as expected currently though? Here's the URL: https://docs.refuel.ai/guide/accuracy/chain-of-thought/

Further, this proposed fix references a URL in `JTarakRam/autolabel` fork

Reply:
Yeah, It is working perfectly Fine !